code/modules/mob/living/carbon/xenomorph/strains/castes/ravager/hedgehog.dm

```diff
@@ -19,9 +19,9 @@
 
 /datum/xeno_strain/hedgehog/apply_strain(mob/living/carbon/xenomorph/ravager/ravager)
 	ravager.plasma_max = 0
-	ravager.small_explosives_stun = FALSE
+	ravager.small_explosives_stun = TRUE
 	ravager.explosivearmor_modifier += XENO_EXPOSIVEARMOR_MOD_SMALL
-	ravager.damage_modifier -= XENO_DAMAGE_MOD_SMALL
+	ravager.damage_modifier -= XENO_DAMAGE_MOD_VERY_SMALL
```

Comment:
I don't think this is necessary to change right now. But we can consider it in the future.